qa-brh.planx-pla.net/manifest.json

```diff
@@ -0,0 +1,60 @@
+{
+  "notes": [
+    "This is the dev environment manifest",
+    "That's all I have to say"
+  ],
+  "jenkins": {
+    "autodeploy": "yes"
+  },
+  "versions": {
+    "ambassador": "quay.io/datawire/ambassador:1.4.2",
+    "aws-es-proxy": "quay.io/cdis/aws-es-proxy:v1.3.1",
+    "arborist": "quay.io/cdis/arborist:2021.11",
+    "dashboard": "quay.io/cdis/gen3-statics:2021.11",
+    "fence": "quay.io/cdis/fence:2021.11",
+    "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.2-debian-cloudwatch",
+    "hatchery": "quay.io/cdis/hatchery:0.2.0",
+    "indexd": "quay.io/cdis/indexd:2021.11",
+    "metadata": "quay.io/cdis/metadata-service:1.6.2",
+    "portal": "quay.io/cdis/data-portal:3.17.4",
+    "peregrine": "quay.io/cdis/peregrine:2021.11",
+    "pidgin": "quay.io/cdis/pidgin:2021.11",
+    "revproxy": "quay.io/cdis/nginx:1.17.6-ctds-1.0.1",
+    "manifestservice": "quay.io/cdis/manifestservice:2021.11",
+    "wts": "quay.io/cdis/workspace-token-service:2021.11",
+    "sheepdog": "quay.io/cdis/sheepdog:2021.11"
+  },
+  "arborist": {
+    "deployment_version": "2"
+  },
+  "indexd": {
+    "arborist": "true"
+  },
+  "global": {
+    "environment": "qa-brh",
+    "hostname": "qa-brh.planx-pla.net",
+    "revproxy_arn": "arn:aws:acm:us-east-1:707767160287:certificate/c676c81c-9546-4e9a-9a72-725dd3912bc8",
+    "dictionary_url": "https://s3.amazonaws.com/dictionary-artifacts/gtexdictionary/4.0.6/schema.json",
+    "dispatcher_job_num": "10",
+    "portal_app": "gitops",
+    "sync_from_dbgap": "False",
+    "kube_bucket": "kube-qaplanetv2-gen3",
+    "logs_bucket": "logs-qaplanetv2-gen3",
+    "useryaml_s3path": "s3://cdis-gen3-users/qa/user.yaml",
+    "tier_access_level": "regular",
+    "tier_access_limit": 50,
+    "public_datasets": true,
+    "netpolicy": "on",
+    "lb_type": "internal"
+  },
+  "metadata": {
+    "USE_AGG_MDS": true,
+    "AGG_MDS_NAMESPACE": "brh"
```

Comment:
should we use qa-brh for this?

Reply:
I was unsure what this namespace refers to, as prod is running in default namespace, therefore I left it as is. 😬